src/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -178,23 +178,54 @@ public static void IgnoredDataShouldNotBeExtensionData()
         }
 
         [Fact]
-        public static void InvalidExtensionValue()
+        public static void ExtensionPropertyObjectValue_Empty()
```

Comment:
While you are updating the tests, can you please also add a test case with `JsonExtensionData` placed on other invalid Dictionary, like `<string, string>` and `<non-string, object>`?

https://github.com/dotnet/corefx/blob/bab7cea4e612c8d0bd13b4a52a2c91ec974f4e96/src/System.Text.Json/tests/Serialization/ExtensionDataTests.cs#L243

Reply:
done